src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1738,12 +1738,17 @@ public virtual void ClearErrorsAndWarnings()
         /// </summary>
         public virtual void ClearInfoMessages()
         {
+            // It is very unlikely a node could be in both info state or persistent info state from the current design
+            // If that exception happens, we should redesign this function or have particular node override the behavior
```

Comment:
are you logging or sending something to analytics when this state occurs?

Reply:
good question, we have no analytics coverage on warning or info yet I believe. Because there could be repetitive warning/infos user did not care or fix, so I was a bit hesitant to log anything info analytics

Reply:
what about just logging something to the console if the following is true:
`It is very unlikely a node could be in both info state and persistent info state`